.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Again, if we're going to do this, we need release notes.

Reply:
in theory, we should be able to keep 3.7 - I think with groups, but just be aware that benchmarking cant be run on 3.7 but eyecite could.

Reply:
I'm not opposed to dropping 3.7, it's got ten months left until it is no longer supported. Just if we drop it, we need to do it properly is all.

Reply:
I restored 3.7 because ... well its got ten months left.